src/main/java/individual/freshplace/entity/Order.java

```diff
@@ -0,0 +1,31 @@
+package individual.freshplace.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+@Getter @Setter
+public class Order extends BaseEntity{
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "order_seq")
+    private Long id;
+
+    @Column(name = "order_status_code")
+    private String status;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "member_seq")
+    private Member member;
+
+    @Embedded
+    private Address address;
+
+    @OneToMany(mappedBy = "order")
+    private List<OrderDetail> orderDetailList;
```

Comment:
```suggestion
    private List<OrderDetail> orderDetails;
```
이미 타입에서 List라는게 명시되어있는데 컬랙션의 네이밍에 List를 붙이는 이유가 있나요?